runtime/compiler/aarch64/runtime/PicBuilder.spp

```diff
@@ -246,7 +259,68 @@ _virtualUnresolvedHelper:
 	hlt	#0	// Not implemented yet
 
 _interfaceCallHelper:
-	hlt	#0	// Not implemented yet
+	stp	x0, x1, [J9SP, #-64]!
+	stp	x2, x3, [J9SP, #16]
+	stp	x4, x5, [J9SP, #32]
+	stp	x6, x7, [J9SP, #48]
+	mov	x3, x30					// preserve LR
+	add	x0, x30, #J9TR_UICSnippet_CP			// get CP/index pair pointer
+	ldr	x1, [x30, #J9TR_ICSnippet_returnAddress]	// get code cache RA
+	mov	x4, x1						// protect RA in x4 (in L_commonLookupException, it is expected)
+	bl	jitResolveInterfaceMethod			// call the helper
+	cbz	x0, L_commonLookupException			// if resolve failed, throw the exception
+	add	x0, x3, #J9TR_Snippet_CallInstruction		// get address of BL instruction in snippet
+	ldr	x1, const_interfaceCompeteSlot2			// get new snippet branch target
```

Comment:
Why is this called `interfaceCompeteSlot2` ?  There's no "competition" among threads as far as I can tell.  It's really just an interface dispatch.  If there isn't any competition, can you rename it to something that won't cause confusion?

Reply:
This label name came from the arm32 version.  I am going to rename it.

Reply:
I would suggest just calling the label `interfaceDispatch` to give a bit more clarity.

Reply:
I cannot find a symbol `interfaceDispatch` in the JIT nor in the VM runtime.
What do you mean by this?

Reply:
I looked into this a bit deeper.  This means:
* renaming the `_interfaceCompeteSlot2` label to `_interfaceDispatch`
* renaming `const_interfaceCompeteSlot2` to `const_interfaceDispatch`, and initializing it with `.dword _interfaceDispatch`
* renaming the helper index `TR_ARM64interfaceCompeteSlot2` to `TR_ARM64interfaceDispatch`
* initializing the `TR_ARM64interfaceDispatch` entry in the helpers table (for example, similar to this entry on ARM): 
 https://github.com/eclipse/openj9/blob/4f639567fd42ca81abab8023687e7942c7d27749/runtime/compiler/runtime/Runtime.cpp#L1431
* deleting `_interfaceSlotsUnavailable` because we don't use it

These are all remnants from a solution copied from Power that isn't used on AArch64 (or ARM for that matter).  If we develop an IPIC scheme for AArch64 then we'll have to architect something then that works best for AArch64.

Reply:
Applied those changes with `_interfaceCompeteSlot2` and `_interfaceSlotsUnavailable`.
It depends on https://github.com/eclipse/omr/pull/4105 for the new symbol `TR_ARM64interfaceDispatch`.